src/Kunstmaan/MediaBundle/Helper/ExtensionGuesserFactory.php

```diff
@@ -2,24 +2,25 @@
 
 namespace Kunstmaan\MediaBundle\Helper;
 
-use Kunstmaan\MediaBundle\Helper\File\SVGExtensionGuesser;
-use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesser;
-use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface;
+use Symfony\Component\Mime\MimeTypes;
+use Symfony\Component\Mime\MimeTypesInterface;
 
+@trigger_error(sprintf('The "%s" class is deprecated since KunstmaanMediaBundle 5.6, use "%s" instead.', ExtensionGuesserFactory::class, MimeTypes::class), E_USER_DEPRECATED);
+
+/**
+ * @deprecated this class is deprecated since KunstmaanMediaBundle 5.6 and will be removed in KunstmaanMediaBundle 6.0
+ */
 class ExtensionGuesserFactory implements ExtensionGuesserFactoryInterface
 {
     /**
      * Should return an extension guesser instance, used for file uploads
      *
      * NOTE: If you override this, you'll probably still have to register the SVGExtensionGuesser as last guesser...
      *
-     * @return ExtensionGuesserInterface
+     * @return MimeTypesInterface
      */
-    public function get()
+    public function get(): MimeTypesInterface
```

Comment:
There should be no changes in this file as it is deprecated. Also the changes here are BC breaks